addons/purchase_stock/tests/test_anglo_saxon_valuation_reconciliation.py

```diff
@@ -53,7 +53,7 @@ def _create_invoice_for_po(self, purchase_order, date):
         move_form.invoice_date = date
         move_form.partner_id = self.partner_a
         move_form.currency_id = self.currency_data['currency']
-        move_form.purchase_id = purchase_order
+        move_form.purchase_vendor_bill_id = self.env['purchase.bill.union'].browse(-purchase_order.id)
```

Comment:
just for curiosity, what does it means the `-` in the `browse(-purchase_order.id)`?